distributed/deploy/spec.py

```diff
@@ -418,7 +418,11 @@ async def _close(self):
 
             await self.scheduler.close()
             for w in self._created:
-                assert w.status in {Status.closed, Status.failed}, w.status
+                assert w.status in {
+                    Status.closing,
+                    Status.closed,
+                    Status.failed,
+                }, w.status
```

Comment:
This is because the scheduler.close does no longer use an RPC to close the remote worker.
This behavior was introduced in https://github.com/dask/distributed/pull/6363